src/main/webapp/i18n/de/exam.json

```diff
@@ -606,6 +610,33 @@
                     "numberParticipants": "Teilnehmende",
                     "variants": "Varianten"
                 }
+            },
+            "suspiciousBehavior": {
+                "title": "Verdächtiges Verhalten",
+                "examSessionDescription": "Bei jedem Eintritt in eine Klausur wird für den Studierenden eine Sitzung erstellt. Diese Sitzung ist die Kombination aus IP-Adresse, Sitzungsschlüssel, User Agent, Browser-Fingerabdruck, Browserinstanz-ID und einem dazugehörigen Zeitstempel.",
+                "suspiciousSessionDescription": "Eine Sitzung wird als verdächtig eingestuft, wenn sie eine der folgenden Kriterien erfüllt:",
+                "suspiciousSessionCriterionIpAddress": "Es gibt mehrere Sitzungen mit derselben IP-Adresse, die nicht derselben Studierendenklausur zugeordnet sind",
+                "suspiciousSessionCriterionBrowserFingerprint": "Es gibt mehrere Sitzungen mit demselben Browser-Fingerabdruck, die nicht derselben Studierendenklausur zugeordnet sind",
+                "suspiciousSessionCriterionUserAgent": "Es gibt mehrere Sitzungen mit demselben User Agent, die nicht derselben Studierendenklausur zugeordnet sind",
+                "suspiciousSessions": {
+                    "title": "Verdächtige Sitzungen",
+                    "view": "Verdächtige Sitzungen anzeigen",
+                    "number": "Diese Klausur hat {{ numberOfSuspiciousSessions }} verdächtige Fälle.",
+                    "reasons": "Verdächtig aufgrund von:",
+                    "case": "Fall",
+                    "sameIpAddress": "Gleiche IP-Adresse",
+                    "sameBrowserFingerprint": "Gleicher Browser-Fingerabdruck",
+                    "sameUserAgent": "Gleicher User Agent"
```

Comment:
Reminder to remove this (only if I understood correctly that the user agent will be completely dropped as a criterium).